policyengine_uk/variables/gov/hmrc/income_tax/liability/dividend_income_tax.py

```diff
@@ -0,0 +1,25 @@
+from policyengine_uk.model_api import *
+
+
+class dividend_income_tax(Variable):
+    value_type = float
+    entity = Person
+    label = "Income tax on dividend income"
+    definition_period = YEAR
+    reference = dict(
+        title="Income Tax Act 2007, s. 13",
+        href="https://www.legislation.gov.uk/ukpga/2007/3/section/13",
+    )
+    unit = GBP
+
+    def formula(person, period, parameters):
+        rates = parameters(period).gov.hmrc.income_tax.rates
+        other_income = person("earned_taxable_income", period) + person(
+            "taxed_savings_income", period
+        )
+        taxable_dividends = person("taxed_dividend_income", period)
+        tax_with_dividends = rates.dividends.calc(
+            other_income + taxable_dividends
+        )
+        tax_without_dividends = rates.dividends.calc(other_income)
+        return tax_with_dividends - tax_without_dividends
```

Comment:
can this be negative?

Reply:
It cannot, and it likely shouldn't be, considering the variable's formula, but I'll add in a `max_()` function to prevent edge-case breakage.